daemon/daemon_unix.go

```diff
@@ -362,7 +362,7 @@ func verifyContainerResources(resources *containertypes.Resources, sysInfo *sysi
 	}
 	if resources.CPUPercent > 0 {
 		warnings = append(warnings, "%s does not support CPU percent. Percent discarded.", runtime.GOOS)
-		logrus.Warn("%s does not support CPU percent. Percent discarded.", runtime.GOOS)
+		logrus.Warnf("%s does not support CPU percent. Percent discarded.", runtime.GOOS)
```

Comment:
oh! overlooked this in https://github.com/docker/docker/pull/23458, thanks for catching :blush: